src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
```

Comment:
Well, there is nothing saying if it is a crossref, string, or alias
resolving at the moment. Hence, the month field will mark gray with content
"Jan" independent of:
* the entry not having a month/date, but the crossref does
* it says #jan# in the field
* the entry has a date xxxx-01-xx

Indeed it may (initially) be a bit confusing for the user when the entry
editor says something different, that is why I wanted a marker. If tooltips
would work in the table one could (in the long run) add a tooltip saying
where the information origins. Also, the idea of having a "resolved"
grayed/italic default text in the entry editor for empty fields may be
worthwhile.